P13-go-to/Program.cs

```diff
@@ -0,0 +1,83 @@
+﻿/*PSEUDOCODE:
+myNumber := random number between 1 and 100
+explain rules to user
+USER_TURN:
+ask user for number
+guess := user input
+
+if myNumber is less than guess then
+   tell user to guess lower
+   go back to USER_TURN
+end if
+
+if myNumber is greater than guess then
+   tell user to guess higher
+   go back to USER_TURN
+end if
+
+if myNumber is equal to guess then
+   tell user he won
+end if*/
+
+/* int gameWin = Random.Shared.Next(100);
+Console.WriteLine("Let's play a game! Guess any whole number between 1 to 100!");
+string userGuess = Console.ReadLine();
+int.Parse(userGuess);
+if (gameWin.Equals(userGuess))
+{
+   Console.WriteLine("That is the right number! Wanna play again?");
+}
+else
+{
+   
+}
+*/
+
+/* Random random = new Random();
+int myNumber = random.Next(1,101);
+
+Console.WriteLine("Welcome to the Number Guessing Game!");
+Console.WriteLine("I have chosen a random number between 1 and 100 and now you have to guess which one it is! Good Luck!");
+
+Console.ReadLine(); */
```

Comment:
It's best to remove comments and/or create separate projects for the other games.